wdio.conf.js

```diff
@@ -3,172 +3,179 @@ const DEBUG = process.env.DEBUG;
+services: [
+'firefox-profile'
+],
+reporters: ['dot', 'spec'],
+//  seleniumArgs: {
+//    drivers: {
+//      chrome: {
+//        version: '2.46',
+//        baseURL: 'https://chromedriver.storage.googleapis.com'
+//      }
+//    }
+//  },
+//  seleniumInstallArgs: {
+//    drivers: {
+//      chrome: {
+//        version: '2.46',
+//        baseURL: 'https://chromedriver.storage.googleapis.com'
+//      }
+//    }
+//  },
+//
+// Framework you want to run your specs with.
+// The following are supported: Mocha, Jasmine, and Cucumber
+// see also: http://webdriver.io/guide/testrunner/frameworks.html
+//
+// Make sure you have the wdio adapter package for the specific framework installed
+// before running any tests.
+framework: 'mocha',
+//
+// Test reporter for stdout.
+// The only one supported by default is 'dot'
+// see also: http://webdriver.io/guide/testrunner/reporters.html
+// reporters: ['dot'],
+//
+// Options to be passed to Mocha.
+// See the full list at http://mochajs.org/
+mochaOpts: {
+timeout: !DEBUG ? 5 * 60 * 1000 : Infinity,
+ui: 'bdd'
+},
+    onPrepare: function() {
+        // eslint-disable-next-line
+        console.log('let\'s go')
+    },
+    onComplete: function() {
+        // eslint-disable-next-line
+        console.log('that\'s it')
     }
-  },
-  seleniumInstallArgs: {
-    drivers: {
-      chrome: {
-        version: '2.46',
-        baseURL: 'https://chromedriver.storage.googleapis.com'
-      }
-    }
-  },
-  //
-  // Framework you want to run your specs with.
-  // The following are supported: Mocha, Jasmine, and Cucumber
-  // see also: http://webdriver.io/guide/testrunner/frameworks.html
-  //
-  // Make sure you have the wdio adapter package for the specific framework installed
-  // before running any tests.
-  framework: 'mocha',
-  //
-  // Test reporter for stdout.
-  // The only one supported by default is 'dot'
-  // see also: http://webdriver.io/guide/testrunner/reporters.html
-  // reporters: ['dot'],
-  //
-  // Options to be passed to Mocha.
-  // See the full list at http://mochajs.org/
-  mochaOpts: {
-    timeout: !DEBUG ? 2 * 60 * 1000 : Infinity,
-    ui: 'bdd'
-  }
 };
 
 if (process.env.SAUCE_USERNAME) {
-  config.services.push('sauce');
-  config.protocol = 'http';
-  config.sauceConnect = true;
-  config.sauceConnectOpts = {
-    tunnelDomains: ['localhost']
-  };
-  config.capabilities.push({
-    browserName: 'chrome',
-    platform: 'macOS 10.12',
-    version: 'latest',
-    screenResolution: '1376x1032',
-    username: process.env.SAUCE_USERNAME,
-    accessKey: process.env.SAUCE_ACCESS_KEY
-  });
-  config.capabilities.push({
-    browserName: 'firefox',
-    platform: 'Windows 8.1',
-    version: 'latest',
-    screenResolution: '1280x960',
-    username: process.env.SAUCE_USERNAME,
-    accessKey: process.env.SAUCE_ACCESS_KEY
-  });
+config.services.push('sauce');
+config.protocol = 'http';
+config.sauceConnect = true;
```

Comment:
Shouldn't we be able to run the regression tests locally without using Sauce Labs?

Reply:
you can run regression tests locally.

Reply:
Ah, the issue isn't Sauce Labs. It's that the test fails in my local version of Firefox because it is v 82.0. Maybe we should remove `browserVersion` from the local test settings.

Reply:
updated in new commit